packages/docs/cookbook/composing-stores.md

```diff
@@ -40,34 +40,12 @@ const useY = defineStore('y', () => {
 })
 ```
 
-## Nested stores
+## Using a store within another store
 
-Note that if one store uses another store, **there is no need to create a new store in a separate file**, you can directly import it. Think of it as nesting.
```

Comment:
This seemed likely to mislead beginners- you do indeed need to define a store in a separate file